src/etc/local_stage0.sh

```diff
@@ -49,6 +49,13 @@ if [ -z $TARG_DIR ]; then
     exit 1
 fi
 
+case "$TARG_DIR" in
+--print-rustc-version)
+  # not actually copying to TARG_DIR, just print the local rustc version and exit
+  ${PREFIX}/bin/rustc${BIN_SUF} --version
```

Comment:
Note that `rustc --version` is never just the same as `CFG_RELEASE` -- it will print something like "rustc RELEASE (optional git info)".  I think `rustc --version --verbose` is better, then you'll see something like:

```
$ rustup run beta rustc --version --verbose
rustc 1.11.0-beta.1 (8dc253bcf 2016-07-05)
binary: rustc
commit-hash: 8dc253bcf3a565deb575108992b9f2e7d985aa17
commit-date: 2016-07-05
host: x86_64-unknown-linux-gnu
release: 1.11.0-beta.1
```

or from my Fedora package in progress:

```
$ /usr/bin/rustc --version --verbose
rustc 1.10.0
binary: rustc
commit-hash: unknown
commit-date: unknown
host: x86_64-unknown-linux-gnu
release: 1.10.0
```

So that last line out can be matched to `CFG_RELEASE`.